Bio/KDTree/KDTreemodule.c

```diff
@@ -1,3 +1,13 @@
+/* Copyright 2008-2019 by Michiel de Hoon.  All rights reserved.
+ *
+ * This file is part of the Biopython distribution and governed by your
+ * choice of the "Biopython License Agreement" or the "BSD 3-Clause License".
+ * Please see the LICENSE file that should have been included as part of this
+ * package.
+ */
+
+
+
```

Comment:
Probably don't need three blank lines here, maybe just one or two?

Reply:
Sure I will fix this. I saw an example and followed it.

Reply:
This is a stylistic judgement, and I'm not up to date of C best practises.